share/spack/gitlab/cloud_pipelines/stacks/ml-linux-power64le-cpu/spack.yaml

```diff
@@ -0,0 +1,142 @@
+spack:
+    #  config:
+    #    install_tree:
```

Comment:
No need for this commented section I think, jobs associated with your stacks should be picking up existing linux/power configs thanks to how your new jobs extend the `.linux-power` and `.base-generate` definitions.

Reply:
Also used for local testing.  Will be removed before PR is finalized.